polaris-service/src/main/java/org/apache/polaris/service/TimedApplicationEventListener.java

```diff
@@ -38,13 +40,21 @@
 @Provider
 public class TimedApplicationEventListener implements ApplicationEventListener {
 
+  public static final String METRIC_NAME = "polaris.TimedApi";
+  public static final String TAG_API_NAME = "API_NAME";
```

Comment:
I think tag names should be lowercase

Reply:
I agree, but we currently uppercase the realm/status tags: https://github.com/apache/polaris/blob/main/polaris-core/src/main/java/org/apache/polaris/core/monitor/PolarisMetricRegistry.java#L45-L46

I'm not sure if we'd want to leave the old tags uppercase or if we want to change them, how we should handle backward compatibility and eventually full deprecation.

Reply:
Lets use lowercase for the new metrics and deprecate the old ones

Reply:
Done! The new tags only emit the `snake_case` version and the old ones emit both. Added lots of comments about deprecation and beefed up the tests.